Dfe.Academies.Academisation.Domain/ProjectAggregate/Project.cs

```diff
@@ -1,29 +1,38 @@
 ﻿using Dfe.Academies.Academisation.Core;
+using Dfe.Academies.Academisation.Domain.ApplicationAggregate.Schools;
 using Dfe.Academies.Academisation.Domain.Core.ApplicationAggregate;
 using Dfe.Academies.Academisation.Domain.Core.ProjectAggregate;
+using Dfe.Academies.Academisation.Domain.SeedWork;
 using Dfe.Academies.Academisation.IDomain.ApplicationAggregate;
 using Dfe.Academies.Academisation.IDomain.ProjectAggregate;
 
 namespace Dfe.Academies.Academisation.Domain.ProjectAggregate;
 
-public class Project : IProject
+public class Project : Entity, IProject, IAggregateRoot
 {
+	protected Project() { }
+
 	private Project(ProjectDetails projectDetails)
 	{
 		Details = projectDetails;
+
 	}
 
+	public IEnumerable<ProjectNote> Notes => _notes.AsReadOnly();
+	IReadOnlyCollection<IProjectNote> IProject.Notes => _notes.AsReadOnly();
+
+	private readonly List<ProjectNote> _notes = new ();
+
 	/// <summary>
 	/// This is the persistence constructor, only use from the data layer
 	/// </summary>
-	public Project(int id, ProjectDetails projectDetails)
+	public Project(int id, ProjectDetails projectDetails, DateTime? createdOn = null)
 	{
 		Id = id;
 		Details = projectDetails;
+		CreatedOn = createdOn ?? DateTime.Now;
```

Comment:
Should we use UTCnow or a different culture info, DateTime.Now as I recall causes some issues with the server being behind by an hour because it's in europe

Reply:
that was what was used everywhere before, if you change to UTC when it is mapped out that will need to be handled, if it was me from fresh everything would be UTC

Reply:
There is probably date issues all over the products, I don't think any are using UTC

Reply:
fair, I remember I did some ToUKTime Shenanigans a while back but that will be client side